lib/ui_components/docu_cop.rb

```diff
@@ -0,0 +1,77 @@
+module UiComponents
+  module DocuCop
```

Comment:
It's also both defined over the `attribute` method. But I will check if I can separate this somewhat cleanly.